static-data/createIdentifiers.cql

```diff
@@ -0,0 +1,6 @@
+// For every industry classification a new UPPIdentifer is created with the value of the uuid of the industry classification
+MATCH (industries:IndustryClassification) 
+WITH COLLECT (industries) as inds
+UNWIND inds as ind
+MERGE (indent:Identifier:UPPIdentifer{value:ind.uuid})-[:IDENTIFIES]->(ind)
```

Comment:
Good catch. I was testing on a copy of prod that didn't have the UPPIdentifier yet!